be/src/tools/meta_tool.cpp

```diff
@@ -1108,6 +1117,24 @@ int meta_tool_main(int argc, char** argv) {
             return -1;
         }
         std::cout << json << '\n';
+    } else if (FLAGS_operation == "lake_datafile_gc") {
+        if (!starrocks::config::init(FLAGS_conf_file.c_str(), false)) {
+            std::cerr << "Init config failed, conf file: " << FLAGS_conf_file << std::endl;
+            return -1;
+        }
+        if (!starrocks::init_glog("lake_datafile_gc", true)) {
+            std::cerr << "Init glog failed" << std::endl;
+            return -1;
+        }
+        Aws::SDKOptions options;
+        Aws::InitAPI(options);
+        auto status =
+                starrocks::lake::datafile_gc(FLAGS_root_path, FLAGS_audit_file, FLAGS_expired_sec, FLAGS_do_delete);
```

Comment:
refuse running if the expire_secs is too small, e.g. less than 5 mins?

Reply:
added a 10min checking